README.rst

```diff
@@ -89,9 +89,12 @@ Follow the setup instructions of `openwisp-controller
     ]
 
     # Make sure you change them in production
-    INFLUXDB_USER = 'openwisp'
-    INFLUXDB_PASSWORD = 'openwisp'
-    INFLUXDB_DATABASE = 'openwisp2'
+    TIMESERIES_DB = {
```

Comment:
I would name it `TIMESERIES_DATABASE` because as per the zen of python, explicit is better than implicit, you may shorten it to `TIMESERIES_DB` when importing, eg: `from ... import TIMESERIES_DATABASE as TIMESERIES_DB`, although unless it's used a lot in the code the explicit form will help readability